flood_adapt/object_model/hazard/event/forcing/waterlevels.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel
+
+from flood_adapt.object_model.hazard.event.forcing.forcing import IForcing
+from flood_adapt.object_model.hazard.event.timeseries import (
+    CSVTimeseriesModel,
+    SyntheticTimeseriesModel,
+)
+from flood_adapt.object_model.io.unitfulvalue import UnitfulLength, UnitfulTime
+
+
+class IWaterlevel(IForcing):
+    pass
+
+
+class SurgeModel(BaseModel):
```

Comment:
I dont see these in site.py so i might be missing something.
They are quite similar (maybe even the same) to the models defined in interface/events.py, but to keep related things together, I thought it would be nice to move these here. Can easily be changes of course